spec/features/users_feature_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+
+feature "User can sign in and out" do
```

Comment:
This is described as a test of signing in and out, but a lot of the motions your going through are for signing up, which is unrelated to the test. In this sort of situation you really want to just inject a user directly into the database. This is part of keeping your tests isolated from each other. Currently, if the functionality of signing up fails, this test will also fail.